cpp/src/structs/utilities.cpp

```diff
@@ -341,6 +341,17 @@ void superimpose_parent_nulls(bitmask_type const* parent_null_mask,
   }
 }
 
+/**
+ * @copydoc cudf::structs::detail::contains_list
+ */
```

Comment:
I'm not sure why a similar copydoc is present in a couple of other places in this file, but to my knowledge only the doxygen in the header is ever parsed and this is superfluous.